### DISCUSSION TITLE
assertViewIs not working in v3

### DISCUSSION THREAD

**Discussion by gbuckingham89** [livewire/livewire#6212] [7. Bugs (archived)] [answered]
### Livewire version

3.0.0-beta.7

### Which PHP version are you using?

PHP 8.2

### Repository URL

https://github.com/gbuckingham89/livewire-v3-assert-view-is

### Steps To Reproduce

1. Create a [component](https://github.com/gbuckingham89/livewire-v3-assert-view-is/blob/master/app/Livewire/CreatePost.php) that renders [a view](https://github.com/gbuckingham89/livewire-v3-assert-view-is/blob/master/resources/views/livewire/create-post.blade.php)
2. Write [a test](https://github.com/gbuckingham89/livewire-v3-assert-view-is/blob/master/tests/Feature/Livewire/CreatePostTest.php) that asserts the component uses a specific view file via `assertViewIs()` 
3. Run the test

The test fails as follows:

```
  FAILED  Tests\Feature\Livewire\CreatePostTest > renders successfully                                       AssertionFailedError   
  The response is not a view.

  at vendor/livewire/livewire/src/Features/SupportTesting/Testable.php:276
    272▕         if (static::hasMacro($method)) {
    273▕             return $this->macroCall($method, $params);
    274▕         }
    275▕ 
  ➜ 276▕         $this->lastState->getResponse()->{$method}(...$params);
    277▕ 
    278▕         return $this;
    279▕     }
    280▕ }

  1   tests/Feature/Livewire/CreatePostTest.php:17


  Tests:    1 failed, 2 passed (3 assertions)
  Duration: 0.15s
```

### Additional information

I checked to see if had been removed, but the docs do mention `assertViewIs()` is available: https://livewire.laravel.com/docs/testing#assertions

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by joshhanley** [comment]
@gbuckingham89 thanks for reporting! Can you submit a failing test PR so we can investigate? Thanks!

**Reply by gbuckingham89** [reply, marked as answer]
Yep, PR added: https://github.com/livewire/livewire/pull/6244
